exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -514,33 +661,100 @@ db.movieDetails.find({"genres":["Action", "Adventure"]},
+## Update Operations
 
-**Activity: :alarm_clock: 15 minutes** 
+The MongoDB query language supports 3 update operations:
+* `updateOne()`
+* `updateMany()`
+* `replaceOne()` - [Read more](https://docs.mongodb.com/manual/reference/method/db.collection.replaceOne/#db.collection.replaceOne) :book: about this operator. 
+
+**Exercise 15** :computer: 
+
+Let's turn our attention towards movies made in 🇮🇳 ! I bet all of you have either heard of or watched "Taare Zameen Par" but unfortunately this collection hasn't got the name right! 
 
-Share with the rest of the class the kind of movies you like by writing 3 interesting queries against this database, at the same time limiting the resulting documents to fields that you find relevant. 
+* Spend **5 minutes** :alarm_clock: to find out what the movie `title` is in our collection! Feel free to use the internet to look up identifying information like the release year or the actor/director/writer names to use in your query filter. 
 
-For example, movies that were part of both the 'Action' and 'Adventure' genres! :boom: 
+* Once you have the name, let's update it to reflect the correct name "Taare Zameen Par". 
 
-Using commands like:
-- `show dbs`
-- `use <database>`
-- `show collections`
+Here's how to perform the update operation:
+```javascript
+db.movieDetails.updateOne(
+    {<your uniquely identifying filter as key: value pair>}, 
+    {"$set": {"title": "Taare Zameen Par"}}
+    )
+```
 
-Explore the different datasets available on your workstation.  Be creative and try to find interesting things.  
+**Exercise 16** :computer: 
 
-Some questions to consider while exploring:
-- How is this data structured?
-- Is there anything interesing to me about this data?
-- Is the data malformed, incorrect or incomplete?
-- What are the challenges you faced while exploring this data?
+MongoDB offers multiple operators that can be used in its update operations. In the exercise above, we looked at the `$set` operator. 
 
-**Presentation: :alarm_clock: 25 minutes**
-Each group will prepare a small, **5 minute**, 2-3 slide presentation about the datasets.
+|Name|Description|
+|---|---|
+|`$set`|Sets the value of a field in a document|
+|`$unset`|Removes the specified field from a document|
+|`$currentDate`|Sets the value of a field to current date, either as a Date or a Timestamp|
+|`$inc`|Increments the value of a field by a specified amount|
+|`$mul`|Multiplies the value of the field by the specified amount|
+|`$min`|Only updates the field if the specified value is less than the existing field value.|
+|`$max`|Only updates the field if the specified value is greater than the existing field value.|
+|`$setOnInsert`|Sets the value of a field if an update results in an insert of a document. Has no effect on update operations that modify existing documents.|
+|`$rename`|Renames a field|
+
+
+Take **10 minutes** :alarm_clock: to go through this list and write an interesting `$updateMany()` query. 
```

Comment:
if you do this you may break the data for future exercises.  it may be better to have them make updates to the employee database that they created in the **Create** section of things.  Having them update the movie name was fine, but a complete overwrite of bulk data can be detrimental.  

Think about how this affects the other exercises and decide if you want to roll these 🎲 or not.  It's your call!